toolkit/package.json

```diff
@@ -38,8 +38,12 @@
   "devDependencies": {
     "@babel/core": "^7.17.0",
     "@babel/eslint-parser": "^7.18.2",
+    "@babel/plugin-proposal-async-generator-functions": "^7.20.7",
     "@babel/plugin-proposal-class-properties": "^7.16.7",
     "@babel/plugin-proposal-decorators": "^7.17.0",
+    "@babel/plugin-proposal-json-strings": "^7.18.6",
+    "@babel/plugin-proposal-object-rest-spread": "^7.20.7",
+    "@babel/plugin-proposal-optional-catch-binding": "^7.18.6",
```

Comment:
Added these because the floating deps install kept failing saying it couldn't find them.
Got the list from: https://github.com/wessberg/rollup-plugin-ts/blob/15e8edd828972cad274caab88ac91081414e82c5/src/transpiler/babel.ts#L109-L117